sdk/core/azure-core/src/main/java/com/azure/core/models/package-info.java

```diff
@@ -2,6 +2,25 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing core model classes.
+ * This package contains the core model classes used across the Azure SDK.
+ *
+ * <p>These classes provide common structures and functionality for working with Azure services. They include
+ * representations for various types of data, such GeoJSON objects, and JSON Patch documents.</p>
+ *
+ * <p>Classes in this package are typically used as base classes or utility classes, and are extended or used by other
+ * classes in the Azure SDK to provide service-specific functionality.</p>
+ *
+ * <p>Some of the key classes in this package include:</p>
+ * <ul>
```

Comment:
order the list so types from similar spaces are grouped together (so Geo next to Json etc)